include/deal.II/distributed/grid_tools.h

```diff
@@ -130,6 +131,21 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> pack,
                                   std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> unpack);
+
+    /**
+     * Compute a vector of bounding boxes which approximates the space occupied by
```

Comment:
I think you want to say "encloses" instead of "approximates". The latter suggests that the inclusion is not exact, but then it would be useless.